### PR TITLE
Update names to Fidesmo App, Fidesmo Service...

### PR DESCRIPTION
**Pull request by miguelcardo**

### CODE REVIEW

**Comment by miguelcardo** [comment]
@petter-fidesmo Ready for review

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Android-Transceive-Demo
 =======================
 
-This repository contains a simple example client showing how to use *Fidesmo Secure Element Connection API* capabilities.
+This repository contains a simple example client showing how to use *Fidesmo API* capabilities.
```
Comment (ghost) [inline README.md]: @miguelcardo I think Fidesmo Service would be more appropriate here. And no "". What do you think?

**Comment by miguelcardo** [comment]
@petter-fidesmo @MattiasEld Corrections done. As soon as you review, I'll squash and you can merge

**Comment by MattiasEld** [comment]
@miguelcardo +1, please squash!

**Comment by miguelcardo** [comment]
@MattiasEld Squashed. Now you can merge!

**Pull request merged**